file-formats/dobj/type/zif_aff_dobj_v1.intf.abap

```diff
@@ -0,0 +1,61 @@
+INTERFACE zif_aff_dobj_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">Structure Definition</p>
+    "! Structure Definition
+    BEGIN OF ty_structure_definition,
```

Comment:
Typically for structures used in a table, we add "... Details to the title and description

```suggestion
    "! <p class="shorttext">Structure Definition Details</p>
    "! Structure definition details
    BEGIN OF ty_structure_definition,
```

Reply:
Title and Description Adapted accordingly